utils/oscap-podman

```diff
@@ -92,14 +93,16 @@ for VAR in `podman inspect $ID --format '{{join .Config.Env " "}}'`; do
     eval "export OSCAP_OFFLINE_$VAR"
 done
 
-export OSCAP_PROBE_ROOT="$(cd "$DIR"; pwd)"
+export OSCAP_PROBE_ROOT
+OSCAP_PROBE_ROOT="$(cd "$DIR" && pwd)"
```

Comment:
ditto

Reply:
Situation here is not really like it is in `oscap-chroot`, but added as well, for sake of completeness.